### ISSUE TITLE
Need to have support case hyperlink 

### ISSUE THREAD

**Issue by anerurka** [jainnikhil30/cases#2]
Need to have a support case hyperlink.
At present, once the cases are categorized we have to search it manually, adding hyperlink will be a good addition.

- Ameya Nerurkar

**Comment by jainnikhil30** [comment]
@anerurka 

I am not sure if we can do this in the spreadsheet with the category details. You can not have multiple hyperlinks for 1 cell (that's a limitation of spreadsheet). We would need the case number in a separate cell for hyper link to be added. In the category spreadsheet all the cases for a particular category are put in one cell only.  

I think the best way to accomplish this would be to have the case link be part of the original data sheet and then you can search for the case there and directly click the link from there ?

**Comment by anerurka** [comment]
@jainnikhil30 , yes that makes sense, let me check if we can have hyperlinks added in the datasource.
